src/ast/codegen_llvm.cpp

```diff
@@ -1190,30 +1190,12 @@ void CodegenLLVM::visit(If &if_block)
 
 void CodegenLLVM::visit(Unroll &unroll)
 {
-  Function *parent = b_.GetInsertBlock()->getParent();
-  BasicBlock *loop = BasicBlock::Create(module_->getContext(), "loop", parent);
-  BasicBlock *done = BasicBlock::Create(module_->getContext(), "done", parent);
-
-  AllocaInst *val = b_.CreateAllocaBPF(SizedType(Type::integer, 8), "loop_count");
-  b_.CreateStore(b_.getInt64(unroll.var), val);
-
-  b_.CreateCondBr(b_.CreateICmpNE(b_.getInt64(unroll.var), b_.getInt64(0), "true_cond"), loop, done);
-
-  b_.SetInsertPoint(loop);
-
-  for (Statement *stmt : *unroll.stmts)
-  {
-    stmt->accept(*this);
+  for (int i=0; i < unroll.var; i++) {
```

Comment:
Unlikely this matters but there might be a compiler warning without it
```suggestion
  for (long int i=0; i < unroll.var; i++) {
```